ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -145,6 +146,12 @@ data RunNodeArgs blk = RunNodeArgs {
       --
       -- Use 'defaultClockSkew' when unsure.
     , rnMaxClockSkew :: ClockSkew
+
```

Comment:
The principle I originally had in mind when we introduced `RunNodeArgs` is to use these fields for parameters for which we *can't* pick default values, e.g., `NetworkMagic`, `ProtocolInfo`, etc. For `NodeArgs` and `ChainDbArgs`, we can, so we provide functions to customise them without having to force the user (i.e., `cardano-node`) to pick the defaults (we know better which defaults to pick).

Over the time `RunNodeArgs` has been changed by multiple people and the principle no longer holds. (I want to refactor/clean up this mess at some point).

That's the reason why I prefer a "customise" function instead of requiring concrete values.